### PR TITLE
Moop enhanced

### PR DESCRIPTION
**Pull request by gjohnson**
Fixed that H1 tag!

### CODE REVIEW

**Comment by thmscly** [comment]
Absolutely brilliant. We're promoting you to super senior. The pay is less, but the pride is also less.

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
   <title>Document</title>
 </head>
 <body>
```
Comment (thmscly) [inline index.html:9]: I think you could have done this better but meh

File: index.html [hunk 2]
```diff
@@ -7,6 +7,6 @@
...
-  <div>Meep<div>
+  <h1>Moop</h1>
```
Comment (thmscly) [inline index.html]:
```suggestion
  <h1>Mooop</h1>
```

**Comment by shimekukuri** [comment]
I question the entire Moop methodology and am personally attacked by it's introduction into the code base.

**Pull request merged**